### PR TITLE
DSD-531: Chakra List

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-531](https://jira.nypl.org/browse/DSD-531)

## This PR does the following:
- Updates the `List` component to use Chakra.
- Updates other references.
- Also fixes a rendering issue with the "Colors" documentation Storybook page (`Breadcrumbs` wasn't updated).
- This is a precursor to updating the `Pagination` component, so updated this first.

## How has this been tested?

Storybook.

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] View [the example in Storybook]()

### CODE REVIEW

File: src/theme/components/list.ts
Comment (bigfishdesign13) [inline src/theme/components/list.ts:19]: To bring this into better alignment with the Figma designs, please make the following changes to the `_before` object.
- add `fontSize: "2"`
- add `lineHeight: "normal"`
- change to `marginLeft: "-.75rem"`
- change to `width: "1rem"`

File: src/theme/components/list.ts
Comment (bigfishdesign13) [inline src/theme/components/list.ts]: The width of the first column was executed incorrectly in the original build of this component.  To fix that error, bump the width value from `225px` up to `250px`.

File: src/theme/components/list.ts
Comment (bigfishdesign13) [inline src/theme/components/list.ts:2]: To bring this into better alignment with the Figma designs, please add the following code to the `baseStyle` object.
```
li: {
  _notFirst: {
    marginTop: "xxs",
  }
},
```

File: src/components/StyleGuide/Colors.stories.mdx
Comment (bigfishdesign13) [inline src/components/StyleGuide/Colors.stories.mdx]: In the updates for the CSS prefix, I removed all of these `List` components and replaced with `SImpleGrid`.  This should make for a fun merge.

**Review by bigfishdesign13 — CHANGES_REQUESTED** [review]
Looks quite good.  Just need to tweak some of the CSS.

File: src/components/StyleGuide/Colors.stories.mdx
Comment (EdwinGuzman) [reply src/components/StyleGuide/Colors.stories.mdx]: I'll make this update after the CSS Prefix pr is merged to make the merge easier.

**Comment by EdwinGuzman** [comment]
Updated all expect the `Colors.stories.mdx` file (will wait until the CSS prefix PR is merged).

File: src/theme/components/list.ts
Comment (bigfishdesign13) [inline src/theme/components/list.ts]: Sorry for additional changes.  This needs to be changed to `lineHeight: "0.9"`.

File: src/theme/components/list.ts
Comment (bigfishdesign13) [inline src/theme/components/list.ts]: Again, sorry for additional changes.  This needs to be changed to `marginLeft: "-1rem"`.

**Review by bigfishdesign13 — CHANGES_REQUESTED** [review]
Sorry, some of the values I gave you yesterday were not correct.  Please see my comments.

**Review by bigfishdesign13 — APPROVED** [review]
This may be cheating because I made the last update, but I am approving this PR.

**Pull request merged**
